chain/client/src/stateless_validation/state_witness_producer.rs

```diff
@@ -422,13 +415,15 @@ impl Client {
     /// Sends the contract accesses to the same chunk validators
     /// (except for the chunk producers that track the same shard),
     /// which will receive the state witness for the new chunk.  
-    fn send_contract_accesses_to_chunk_validators(
+    fn send_contract_updates_to_validators(
```

Comment:
Seems like we are still only sending contract_accesses and not contract_updates. Do we want to rename this function?

Reply:
I am planning to also send the deployments, but yeah I can rename in the later PRs. Reverted the naming change here.